Dockerfile.windows

```diff
@@ -252,6 +255,16 @@ RUN `
   Remove-Item C:\binutils.zip; `
   Remove-Item C:\gitsetup.zip; `
   `
+  Write-Host INFO: Downloading containerd; `
+  Install-Package -Force 7Zip4PowerShell; `
```

Comment:
Hm.. we should ask them to upload `.zip` for Windows.

(But should no longer be an issue if we start uploading containerd binaries to download.docker.com)

Reply:
Huh, just noticed this. Windows has included `tar` for a long time, can we not just use that directly?

Reply:
TIL! (It's clearly been a while since I worked on Windows). Yes if we don't need to install, that'd be great.

Reply:
Windows Server 2016 does not include `tar` so win-RS1 (which is still enabled on master branch builds) would stop working if we add requirement for it.

Reply:
Updated but looks that I still missed couple of `ContainerD` texts which still need to be updated and Win 2022 without containerd did hit #42612 but what we want to do with this one?

Options are:
1) stay on 7zip
2) switch to `tar` and add logic to Dockerfile that containerd will be only installed if OS version is at least RS5

Reply:
Would an equivalent of a `command -v tar` work? (check if the command exists, and if not, download it)? A quick Google search brought me to this page; https://www.shellhacks.com/windows-which-equivalent-cmd-powershell/, and a stackoverflow thread; https://superuser.com/questions/34492/powershell-equivalent-to-the-unix-which-command


I'm ok with doing it in a follow-up if it's too complicated though

Reply:
The win-RS1 parallel in the build is disabled or inactive, last time I looked, and I recall someone trying it briefly in one of the related PRs, and discovering it to be non-working, or at least needs a bunch of tests skipped.

Not that I'm advocating breaking it further, but there was a related discussion about dropping support in the 22.x release, since Server LTSC2016 falls out of mainstream support in January 2022 anyway.

Reply:
RS1 still run from master branch _after_ PR is merged to master. You can see build results by browsing commit history https://github.com/moby/moby/commits/master and clicking that green dot / red x from there. Also afaik those builds are still used as part of Docker EE packaging.

Reply:
But CI green 🟢 so maybe we go with this one now?